src/helm/proxy/clients/huggingface_model_registry.py

```diff
@@ -97,7 +159,30 @@ def register_huggingface_model_config(model_name: str) -> HuggingFaceModelConfig
         name=model_name,
         tags=[TEXT_MODEL_TAG, FULL_FUNCTIONALITY_TEXT_MODEL_TAG],
     )
-    MODEL_NAME_TO_MODEL[model_name] = model
+    MODEL_NAME_TO_MODEL[config.model_id] = model
+    ALL_MODELS.append(model)
+    hlog(f"Registered Hugging Face model: {model} config: {config}")
+    return config
+
+
+def register_huggingface_local_model_config(path: str) -> HuggingFaceLocalModelConfig:
+    """Register a AutoModelForCausalLM model from a local directory for later use.
+
+    path: a path to your HF model"""
+    config = HuggingFaceLocalModelConfig.from_path(path)
+    if config.model_id in _huggingface_model_registry:
+        raise ValueError(f"A Hugging Face model is already registered for model_id {model_name}")
+    _huggingface_model_registry[config.model_id] = config
+
+    if config.model_name in MODEL_NAME_TO_MODEL:
+        raise ValueError(f"A HELM model is already registered for model name: {model_name}")
```

Comment:
These should be `config.model_id` instead of `config.model_name` and `model_name`.